fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -338,16 +338,51 @@ public class Config extends ConfigBase {
      */
     @ConfField public static long max_bdbje_clock_delta_ms = 5000; // 5s
 
+    /**
+     * Whether to enable all http interface authentication
+     */
+    @ConfField public static boolean enable_all_http_auth = false;
+
     /**
      * Fe http port
      * Currently, all FEs' http port must be same.
      */
     @ConfField public static int http_port = 8030;
 
     /**
-     * Whether to enable all http interface authentication
+     * Fe https port
+     * Currently, all FEs' https port must be same.
      */
-    @ConfField public static boolean enable_all_http_auth = false;
+    @ConfField public static int https_port = 8050;
```

Comment:
why need a seperate https port? I think https just on the original http port, and maybe it is called enable_https.

Reply:
Current version of springboot does not support http and https sharing same port.

Reply:
enable_all_http_auth and enable_https are two different parameters.